website/client/components/inventory/stable/index.vue

```diff
@@ -461,6 +464,12 @@
         let search = this.searchText.toLowerCase();
         this.searchTextThrottled = search;
       }, 250),
+      selectedSortBy: {
+        handler () {
+          localStorage.setItem('selectedSortBy', JSON.stringify(this.selectedSortBy));
```

Comment:
JSON.stringify isn't necessary since we already have a string, it's only used when you want to transform an object into a json string